src/test/java/teammates/test/cases/storage/InstructorsDbTest.java

```diff
@@ -314,12 +314,20 @@ public void testUpdateInstructorByGoogleId() throws Exception {
 
         instructorToEdit.name = "New Name";
         instructorToEdit.email = "InstrDbT.new-email@email.tmt";
+        instructorToEdit.isArchived = true;
+        instructorToEdit.role = "new role";
+        instructorToEdit.isDisplayedToStudents = false;
+        instructorToEdit.displayedName = "New Displayed Name";
         instructorsDb.updateInstructorByGoogleId(instructorToEdit);
```

Comment:
This is what a `privelegesAsText` string would look like:
```JAVA
{
  "courseLevel": {
    "canviewstudentinsection": true,
    "cansubmitsessioninsection": true,
    "canmodifysessioncommentinsection": true,
    "canmodifycourse": false,
    "canviewsessioninsection": true,
    "canmodifysession": true,
    "canmodifystudent": true,
    "canmodifyinstructor": true
  },
  "sectionLevel": {},
  "sessionLevel": {}
}
```

Creating such a large string in the method itself would not read very well in my opinion. How would you suggest changing this? (or is there another method to edit privileges that I am missing?O)